RecoEgamma/EgammaIsolationAlgos/python/egammaHBHERecHitThreshold_cff.py

```diff
@@ -7,11 +7,20 @@
     recHitEThresholdHB = _thresholdsHBphase1,
     recHitEThresholdHE = _thresholdsHEphase1,
     maxHcalRecHitSeverity = cms.int32(9),
+    usePFThresholdsFromDB = cms.bool(False)
 )
 
-egammaHBHERecHit_2023 = egammaHBHERecHit.clone(
-    recHitEThresholdHB = _thresholdsHBphase1_2023
-)
+#egammaHBHERecHit_2023 = egammaHBHERecHit.clone(
+#    recHitEThresholdHB = _thresholdsHBphase1_2023
+#)
+
+# from Configuration.Eras.Modifier_run3_egamma_2023_cff import run3_egamma_2023
+# run3_egamma_2023.toReplaceWith(egammaHBHERecHit,egammaHBHERecHit_2023)
+
+from Configuration.Eras.Modifier_run3_egamma_cff import run3_egamma
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+from Configuration.Eras.Modifier_run3_hcal_cff import run3_hcal
+run3_hcal.toModify(egammaHBHERecHit,
```

Comment:
In https://github.com/cms-sw/cmssw/pull/43025 a different modifier was used `hcalPfCutsFromDB`. I think it should be used uniformly across areas.